tests/foreman/cli/test_contentview.py

```diff
@@ -4794,6 +4794,29 @@ def test_positive_incremental_update_propagate_composite(self):
         )
 
 
+    @tier3
+    def test_positive_katello_reimport(self):
+        """ Close loop bug for running katello:reimport.  Making sure
+        that katello:reimport works and doesn't throw an error.
+
+        :id: b4119265-1bf0-4b0b-8b96-43f68af39708
+
+        :Steps: Have satellite up and run 'foreman-rake katello:reimport'
+
+        :expectedresults: Successfully reimport without errors
+
+        :CaseAutomation: automated
+
+        :CaseImportance: Medium
+
+        :bz: 1771555
+        """
+
+        result = ssh.command('foreman-rake katello:reimport')
```

Comment:
There is no setup needed? Just run `foreman-rake` on freshly installed Satellite and that's it?

I'm not saying it's wrong, just a little hard to believe, so I would like to double-check :) .

Reply:
yup!  That's literally it!